src/Fantomas/CodePrinter.fs

```diff
@@ -1792,8 +1818,11 @@ and genExpr astContext synExpr =
 
     | InfixApp (operatorText, operatorExpr, e1, e2) ->
         fun ctx ->
-            isShortExpression
-                ctx.Config.MaxInfixOperatorExpression
+            let size =
```

Comment:
Does this not miss any further check whether the infix operator is a multiline one?